### PR TITLE
add information on how to build the plugins to the README

### PR DESCRIPTION
**Pull request by av500**
Signed-off-by: Vladimir Pantelic <vladimir@pantelic.net>

### CODE REVIEW

**Comment by Be-ing** [comment]
Might it be a better idea to add each of the module repositories as submodules?

**Comment by av500** [comment]
maybe, but people will still have their own plugin repositories that they need to build

**Comment by AndrewBelt** [comment]
Plugin repos should be entirely separate. I know it's a hassle, but third party and my own commercial plugins will come later. I could add some `make dist-plugins`, `make plugins`, etc phony targets in the Makefile that scan the directory and run recursive make.

Also I don't even like using submodules for the `ext/` directory. git implements it in an annoying, nonintuitive way.

**Comment by Be-ing** [comment]
I think there should at least be a stub "plugins" directory with a README file.

**Comment by oblivionratula** [comment]
av500, thank you SO much for the instructions!

**Comment by AndrewBelt** [comment]
8ea37a3

File: README.md [hunk 1]
```diff
@@ -25,6 +25,39 @@ Build Rack.
 
 	make
 
+## Building the plugins
+
+create a subbdirectory "plugins" and change into it:
+
+	mkdir plugins
```
Comment (Be-ing) [inline README.md]: This line should be removed now.

**Comment by AndrewBelt** [comment]
Closing with 5b226794e0ddaab61db6c72cf0f38516200242aa

**Pull request closed without merging**